picard/util/tags.py

```diff
@@ -166,652 +227,1128 @@ def names(self, selector=None):
         is_file_info=True,
         is_hidden=True,
         is_tag=False,
+        not_from_mb=True,
     ),
     TagVar(
         'format',
         shortdesc=N_('File format'),
+        longdesc=N_('Media format of the file (e.g.: MPEG-1 Audio).'),
         is_file_info=True,
         is_hidden=True,
         is_preserved=True,
         is_tag=False,
+        not_from_mb=True,
     ),
     TagVar(
         'gapless',
         shortdesc=N_('Gapless Playback'),
+        longdesc=N_('Indicates whether or not there are gaps between the recordings on the release.'),
+        not_from_mb=True,
     ),
     TagVar(
         'genre',
         shortdesc=N_('Genre'),
+        longdesc=N_(
+            'A multi-value tag containing the specified genre information from MusicBrainz. Requires the '
+            '"Use genres from MusicBrainz" option setting be enabled.'
+        ),
     ),
     TagVar(
+        # TODO: Check if this actually exists or if it was provided by the last.fm plugin.
         'grouping',
         shortdesc=N_('Grouping'),
+        longdesc=N_(''),
+        not_from_mb=True,
     ),
     TagVar(
         'isrc',
         shortdesc=N_('ISRC'),
+        longdesc=N_(
+            'The International Standard Recording Code - an international standard code for uniquely '
+            'identifying sound recordings and music video recordings.'
+        ),
     ),
     TagVar(
         'key',
         shortdesc=N_('Key'),
+        longdesc=N_('The key of the music.'),
+        not_from_mb=True,
     ),
     TagVar(
         'label',
         shortdesc=N_('Record Label'),
+        longdesc=N_('A multi-value tag containing the names of the labels associated with the release.'),
     ),
     TagVar(
         'language',
         shortdesc=N_('Language'),
+        longdesc=N_('Work lyric language as per ISO 639-3 if a related work exists.'),
     ),
     TagVar(
         'length',
         shortdesc=N_('Length'),
+        longdesc=N_('The length of the track in format mins:secs.'),
         is_hidden=True,
         is_tag=False,
     ),
     TagVar(
         'license',
         shortdesc=N_('License'),
+        longdesc=N_('The licenses associated with the track, either through the release or recording relationships.'),
     ),
     TagVar(
         'lyricist',
         shortdesc=N_('Lyricist'),
+        longdesc=N_('The names of the lyricists for the associated work.'),
+    ),
+    TagVar(
+        'lyricistsort',
+        shortdesc=N_('Lyricist Sort'),
+        longdesc=N_('The sort names of the lyricists for the associated work.'),
+        is_hidden=True,
     ),
     TagVar(
         'lyrics',
         shortdesc=N_('Lyrics'),
+        longdesc=N_('The lyrics for the track.'),
+        not_from_mb=True,
     ),
     TagVar(
         'media',
         shortdesc=N_('Media'),
+        # TODO: Confirm this is a multi-value.
+        longdesc=N_('A multi-value tag containing the media on which the release was distributed (e.g.: CD).'),
```

Comment:
This isn't multi-value. This is set per track, and a track only has a single media type